contracts/upgradeable_contracts/native_to_erc20/HomeBridgeNativeToErc.sol

```diff
@@ -44,6 +44,11 @@ contract HomeBridgeNativeToErc is EternalStorage, BasicBridge, BasicHomeBridge {
         require(msg.data.length == 0);
         require(withinLimit(msg.value));
         setTotalSpentPerDay(getCurrentDay(), totalSpentPerDay(getCurrentDay()).add(msg.value));
+        incrementNonce(msg.sender);
```

Comment:
I don't like an idea that we are increasing gas consumption significantly (40 000) whereas the issue for Foreign->Home transactions still exists.
I would suggest to not address these changes at all since in order to implement actual attack:
1. connection must be spoofed for every bridge instance
AND
2. HTTP needs to be configured for RPC channel explicitly instead of HTTPS

Reply:
we agreed to make those changes and now what? going back again  to your decision? what the hell is going on?